drivers/remote/driver.go

```diff
@@ -229,8 +231,19 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 			if jinfo.AddStaticRoute(route.Destination, route.RouteType, route.NextHop) != nil {
 				return errorWithRollback(fmt.Sprintf("failed to set static route: %v", route), d.Leave(nid, eid))
 			}
+			if route.Destination.String() == "0.0.0.0/0" {
+				defaultRoute++
+			}
 		}
 	}
+	jinfo.DisableGatewayService(res.DisableGatewayService)
+	if !res.DisableGatewayService {
+		defaultRoute++
+		log.Infof("enabled libnetwork default gateway service")
+	}
+	if defaultRoute > 1 {
```

Comment:
Wont this be true always for the bridge driver (or) any other existing driver that always returns a valid `res.Gateway` and `res.DisableGatewayService=false` ?
Though your intention is valid, we have to make sure all the existing drivers must work seamlessly without the need to set or reset the newly added flag `DisableGatewayService`.